src/Oscoin/Consensus/Config.hs

```diff
@@ -21,13 +21,18 @@ configPath = "data/config.yaml"
 
 -- | Consensus configuration parameters.
 data Config = Config
-    { maxBlockSize :: !Int
+    { maxBlockSize     :: Int
     -- ^ The maximum block size in bytes.
+    , mutableChainSize :: Int
+    -- ^ The size, in blocks, of the \"mutable part\" of the chain,
+    -- i.e. the part of the chain still subject to rollbacks. In other words
+    -- this is the size of the not-yet-fully-adopted portion of the chain.
```

Comment:
I'd use `Depth` here for the type and name: `mutableChainDepth :: Depth` for clarity. I think the last sentence of the docstring can also be removed, as it introduces a new concept ("adopted") which is a bit confusing imo.